Assignment2/amg080-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,49 @@
+{
+  "@context":{
+    "exp": "http://example.org/class",
```

Comment:
When defining a prefix please remember that the URI must end with / or # depending on the vocabulary.